sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitTestResultImporter.java

```diff
@@ -19,15 +19,24 @@
  */
 package org.sonar.plugins.php.phpunit;
 
-import org.junit.Test;
+import com.google.common.base.Preconditions;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.base.Preconditions'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)